sdk/servicebus/service-bus/samples/javascript/README.md

```diff
@@ -11,7 +11,7 @@ These sample programs show how to use the JavaScript client libraries for Azure
 | [receiveMessagesLoop.js][receivemessagesloop]                       | uses the receiveMessages() function to receive Service Bus messages in a loop                                                  |
 | [scheduledMessages.js][scheduledmessages]                           | uses the scheduleMessage() function to schedule messages to appear on a Service Bus Queue/Subscription at a later time         |
 | [session.js][session]                                               | sends/receives messages to/from session enabled queues/subscriptions in Service Bus                                            |
-| [browseMessages.js][browsemessages]                                 | uses the browseMessages() function to browse a Service Bus                                                                     |
+| [browseMessages.js][browsemessages]                                 | uses the peekMessages() function to browse a Service Bus                                                                     |
```

Comment:
I havent changed the file name in samples on purpose in order to avoid broken links.

Reply:
The funny thing is I believe that was actually the original filename even when it was called peek :)

Reply:
lol